### ISSUE TITLE
Velocity VL Loop

### ISSUE THREAD

**Issue by Foulwerp** [konsolas/AAC-Issues#567]
### Issue description
EDIT
Seems like the VL loop happens when taking any kind of damage just happened when I was in combat.
END EDIT

Jumping and taking fall damage and not moving causes velocity violations. When you reach roughly 6 violations it puts you in an endless loop racking up violations and teleporting the player. This lasts until you get kicked or leave the server and rejoin. 

http://i.imgur.com/tTH796q.mp4

New Video showing it happens with just one jump also. Just a sprint jump and landing. This also has teleport disabled in the configs.

http://i.imgur.com/QMK5ywq.mp4

```# Velocity makes sure that players take proper knockback
velocity:
   # Do you want to enable the velocity check
   enabled: true

   # How many ticks should AAC check a player for? Higher values compensate for higher ping.
   check_length: 15

   # What is the maximum distance AAC should teleport a detected player?
   # If this is zero, the detected player will not be teleported at all
   max_tp_dist: 3.0

   # How many violations before AAC starts teleporting players who ignore velocity?
   min_vl: 0

   # How much should AAC increase a player's violation level by on detection?
   vl_normal: 2

   # If a player seems frozen (e.g. changing resource packs/lagging), how much should AAC increase their violation level?
   # 0 by default to prevent false positives with changing resource packs
   vl_no_packet: 0

   # Map of violation levels to commands
   thresholds:
      #6: "aacstaffnotify {player} seems to be ignoring velocity"
      #12: "aackick {player} Not taking knockback"
```

### Core information
**Server version**: Paper version git-Paper-1081 (MC: 1.11.2) (Implementing API version 1.11.2-R0.1-SNAPSHOT)

**AAC version**: 3.1.4-b1

**ProtocolLib version**: 4.2.1

**Plugin list**: ProtocolLib, AAC

### Special environment information

**ViaVersion version**: None

**ProtocolSupport version**: None

**ViaBackwards version**: None

**BungeeCord plugins**: None

**Comment by Janmm14** [comment]
Temporary "fix" would be to disable the velocity check and use the old knockback check instead.

**Comment by konsolas** [comment]
Default min_vl is 5, but will look at this.

**Comment by konsolas** [comment]
Fixed in 3.1.4-b2
